pkg/security/resolvers/sbom/resolver.go

```diff
@@ -183,7 +187,13 @@ func (r *Resolver) Start(ctx context.Context) {
 				return
 			case sbom := <-r.scannerChan:
 				if err := r.analyzeWorkload(sbom); err != nil {
-					seclog.Errorf("couldn't scan '%s': %v", sbom.ContainerID, err)
+					go func() {
```

Comment:
we are already in a separate goroutine, why do we need to spawn another one ? this could allow to include the first try in the `retry.Do`